qiskit_nature/operators/second_quantization/qubit_converter.py

```diff
@@ -69,6 +73,11 @@ def __init__(self,
                 sector can be provided (a list of int of values -1, and 1). The default is None
                 meaning no symmetry reduction is done.
         """
+        warnings.warn('This QubitConverter is deprecated as of 0.1.0, '
+                      'and will be removed no earlier than 3 months after the release. '
```

Comment:
Also, since this deprecation string always has the same general structure maybe we should think about extracting that into a wrapper method which takes only a few arguments (class name, version number, new location, etc.)

Just an FYI...

Reply:
@manoelmarques I know Manoel had done something like that for Aqua before. Maybe it would be useful to have in the repos in general to facilitate things. But this can be added to master at any time.